### DISCUSSION TITLE
卡在御魂结算页面了

### DISCUSSION THREAD

**Discussion by q147258134** [lisai9093/YYS#17] [Q&A] [answered]
![1719451724066](https://github.com/lisai9093/YYS/assets/25542139/87fe7a13-33bf-4ccc-83fe-f41e49a355cd)
脚本执行卡在上图执行的位置，请问我应该覆盖修改哪一张图片文件呢，截取的应该是哪一块？

**Comment by lisai9093** [comment, marked as answer]
你可以把“战斗结算个性化” 关闭就不会显示这个界面了：
<img width="608" alt="image" src="https://github.com/lisai9093/YYS/assets/3047171/ba4185fa-255b-41a9-b21a-411b66843f41">
